File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution(object):
    def shuffle(self, nums, n):
        """
        :type nums: List[int]
        :type n: int
        :rtype: List[int]
        """
        res=[0]*(2*n)
        for i in range(0,2*n,2):
            res[i]=nums[i//2]
            res[i+1]=nums[i//2+n]
        return res

File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector <int> res(2*n,0);
        int x=0;
        int y=1;
        for(int i=0;i<n;i++){
            res[x]=nums[i];
            res[y]=nums[n+i];
            x+=2;
            y+=2;
        }
        return res;
    }
};